101-cognitive-services-translate/azuredeploy.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "AccountsName": {
+      "defaultValue": "translate",
```

Comment:
@bmoore-msft  no, it could be not globally unique, because it's only visible for you in your subscription.